man/systemd-analyze.xml

```diff
@@ -818,11 +818,13 @@ $ systemd-analyze verify /tmp/source:alias.service
 
         <listitem><para>Control verification of units and their dependencies and whether
         <command>systemd-analyze verify</command> exits with a non-zero process exit status or not. With
+        <command>compat</command>, return zero as the exit status regardless of whether warnings arise during
```

Comment:
Bikeshedding: call it 'never' instead? Ie, describe what it does, not why it's there